### ISSUE TITLE
jc.getWork always returned: []

### ISSUE THREAD

**Issue by tilumi** [vsivsi/meteor-job-collection#234]
I have following small sample for testing `jc.getWork`.It is wired that I always get empty result from `getWork`. I wonder is there any problem in this sample?

```
let jobQueue = new JobCollection('jobQueue');
jobQueue.setLogStream(process.stdout);
if (Meteor.isServer) {
  jobQueue.allow({
    admin: true
  });
}
let jobTypes = {
  convertAndInsert: 'convertAndInsert'
};

if (Meteor.isServer) {
  var worker = function (job, cb) {
    job.done();
    return cb();
  };
  new Job(jobQueue, jobTypes.convertAndInsert, {
    file: "XDDD"
  }).delay(0).save();
  console.log(jobQueue.getWork(jobTypes.convertAndInsert));

}

```

**Comment by tilumi** [comment]
I found I missed `jobQueue.startJobServer();`, so close the issue.

**Comment by vsivsi** [comment]
@tilumi  Glad to figured it out.  

You may already know this, but `jc.getWork()` is a pretty low level function.  You should check out `jc.processJobs()` first to see if it satisfies your requirements.  `jc.processJobs()` is build on top of `jc.getWork()`.

**Comment by tilumi** [comment]
Yeah, at first I use `jc.processJobs()` and cannot get jobs to run, so I create this simplest program to debug. 
I think throwing an error when calling `jc.processJobs();` or `jc.getWork` if `jc.startJobServer();` has not been called is much better for debugging.

**Comment by vsivsi** [comment]
Where would the throw happen? It's not a fatal error, and there's nothing to be done to handle it. There simply isn't any work available before the server is started.

The current behavior is by design, allowing a server to be gracefully shutdown and restarted (e.g. to perform maintenance) without all of the (potentially hundreds of) workers also needing to be restarted.

**Comment by tilumi** [comment]
OK, throwing exceptions may inappropriate, but I think a mechanism to acknowledge caller that the server is not started is better.
